__tests__/app.test.js

```diff
@@ -24,4 +24,27 @@ describe("GET /api", () => {
         expect(endpoints).toEqual(endpointsJson);
       });
   });
-});
\ No newline at end of file
+});
+
+describe("GET /api/topics", () => {
+  test("200: Responds with all topics, with all topics containing correct properties", () => {
+    return request(app)
+    .get("/api/topics")
+    .expect(200)
+    .then(({body: { topics }}) => {
+      expect(topics).toHaveLength(3)
+      topics.forEach((topic) => {
+        expect(topic).toMatchObject({
+          slug: expect.any(String),
+          description: expect.any(String)
+        })
+      });
+    })
+  })
+
+  test("404: Responds with error if user does not input address correctly", () => {
+    return request(app)
+    .get("/api/topicz")
+    .expect(404)
```

Comment:
This test is great, but it's not exclusive to the `/api/topics` endpoint. As the user could force this 404 by putting **any** incorrect address, it should be moved to it's own describe block 👍